x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/DateHistoGroupConfig.java

```diff
@@ -159,7 +159,7 @@ public Rounding createRounding() {
         vsBuilder.dateHistogramInterval(interval);
         vsBuilder.field(field);
         vsBuilder.timeZone(timeZone);
-
+        vsBuilder.missingBucket(true);
```

Comment:
We shouldn't allow missing bucket here. We use this config for the timestamp field which a required field for rollup documents. Moreover since it's the first field in the composite, enabling missing buckets would disable the composite optimization when the field is indexed. IMO it's ok to require the timestamp field to be present since otherwise we have no way to know if the document should be part of the rollup.

Reply:
Ah yeah, silly me.  Absolutely shouldn't allow missing on the timestamp field :)

Good catch thanks